src/Connectors/test/CloudFoundry.Test/CloudFoundryServiceInfoCreatorTest.cs

```diff
@@ -66,353 +64,4 @@ public void BuildServiceInfos_NoCloudFoundryServices_BuildsExpected()
-                        ""label"": ""mongodb-odb"",
-                        ""provider"": null,
-                        ""plan"": ""replica_set"",
-                        ""name"": ""autosource_vehicle_service_mongodb"",
-                        ""tags"": [""mongodb""]
-                    }],
-                    ""p-service-registry"": [{
-                        ""credentials"": {
-                            ""uri"": ""https://eureka-a015d976-af2e-430c-81f6-4f99272ccd24.apps.preprdpcf01.foo.com"",
-                            ""client_secret"": ""foo"",
-                            ""client_id"": ""p-service-registry-bd61a360-f39a-45f8-b022-bbb"",
-                            ""access_token_uri"": ""https://p-spring-cloud-services.uaa.sys.preprdpcf01.foo.com/oauth/token""
-                        },
-                        ""syslog_drain_url"": null,
-                        ""volume_mounts"": [],
-                        ""label"": ""p-service-registry"",
-                        ""provider"": null,
-                        ""plan"": ""standard"",
-                        ""name"": ""autosource_service_registry"",
-                        ""tags"": [
-                            ""eureka"",
-                            ""discovery"",
-                            ""registry"",
-                            ""spring-cloud""
-                        ]
-                    }],
-                    ""p-config-server"": [{
-                        ""credentials"": {
-                            ""uri"": ""https://config-86e87517-cf1f-4112-af74-c9c7c957e7df.apps.preprdpcf01.foo.com"",
-                            ""client_secret"": ""foo"",
-                            ""client_id"": ""p-config-server-4c82e877-1c3c-4029-a7cc-4886ae3f444"",
-                            ""access_token_uri"": ""https://p-spring-cloud-services.uaa.sys.preprdpcf01.foo.com/oauth/token""
-                        },
-                        ""syslog_drain_url"": null,
-                        ""volume_mounts"": [],
-                        ""label"": ""p-config-server"",
-                        ""provider"": null,
-                        ""plan"": ""standard"",
-                        ""name"": ""autosource_config_server"",
-                        ""tags"": [
-                            ""configuration"",
-                            ""spring-cloud""
-                        ]
-                    }],
-                    ""user-provided"": [{
-                        ""credentials"": {
-                            ""uri"": ""10.66.42.50:9001"",
-                            ""accesskey"": ""foo"",
-                            ""secretkey"": ""bar""
-                        },
-                        ""syslog_drain_url"": """",
-                        ""volume_mounts"": [],
-                        ""label"": ""user-provided"",
-                        ""name"": ""foo_minio"",
-                        ""tags"": []
-                    }]
```

Comment:
This question probably goes beyond the scope of the work you've been doing, and it might not be quite the right location to raise it, but when I asked (outside of this PR) about CUPS support, I was referring to whether we might be able to recognize and support a binding that looks like this:

```
                   ""user-provided"": [{
                        ""credentials"": {
                            ""uri"": ""amqp://si_username:si_password@localhost:5672/si_vhost""
                        },
                        ""syslog_drain_url"": """",
                        ""volume_mounts"": [],
                        ""label"": ""user-provided"",
                        ""name"": ""foo_minio"",
                        ""tags"": []
                    }]
```

Reply:
Thanks for the pointer. Let's revisit this during the design of extensibility for connectors.